### PR TITLE
Connect the import operator with analyzer plugins

### PR DESCRIPTION
**Pull request by dominiklohmann**
This changes the import operator to also connect to all analyzer plugins, thus restoring the functionality of the matcher and inventory plugins when using the import operator rather than the import command.

### CODE REVIEW

**Review by tobim — APPROVED** [review]
I think we want this until the existing analyzers are rewritten in a pipeline-native way.

File: libvast/builtins/operators/import.cpp [hunk 1]
```diff
@@ -87,6 +87,9 @@ class import_operator final : public crtp_operator<import_operator> {
     auto source = caf::detail::make_stream_source<import_source_driver>(
       &ctrl.self(), input, num_events, ctrl);
     source->add_outbound_path(index);
+    for (const auto& analyzer : plugins::get<analyzer_plugin>()) {
+      source->add_outbound_path(analyzer->analyzer());
```
[the remark this answers is not in the document]
Comment (dominiklohmann) [reply libvast/builtins/operators/import.cpp:91]: Would you be fine with an assertion as well? The pipeline API should guarantee that remote operators only run after the node is initialized, so this should be guaranteed. I can add a comment explaining this as well.

[the remark this answers is not in the document]
Comment (dominiklohmann) [reply libvast/builtins/operators/import.cpp]: See f3b9c1dc92

[the remark this answers is not in the document]
Comment (tobim) [reply libvast/builtins/operators/import.cpp]: Good to know.

**Pull request merged**